packages/sdk/typescript/subgraph/schema.graphql

```diff
@@ -15,6 +15,18 @@ type Worker @entity {
   payoutCount: BigInt!
 }
 
+type UniqueSender @entity {
```

Comment:
You don't need these new entities. You can check hmt transfer events of the day to know if the this wallet was already a sender or receiver.